src/LinuxEvent.Tests/EventParseTests.cs

```diff
@@ -95,6 +93,16 @@ public void OneStack()
 			});
 		}
 
+        [Fact]
+        [UseCulture("fr-FR")]
```

Comment:
Yes, I ran the test without the change on double.Parse but with the preservation of the culture. The test failed. Setting the culture to **CultureInfo.InvariantCulture** in _double.Parse_ make the test green.